src/java/org/apache/cassandra/tcm/PaxosBackedProcessor.java

```diff
@@ -87,7 +88,8 @@ public ClusterMetadata fetchLogAndWait(Epoch waitFor, Retry.Deadline retryPolicy
             {
                 JVMStabilityInspector.inspectThrowable(t);
                 TCMMetrics.instance.fetchCMSLogConsistencyDowngrade.mark();
-                logger.warn("Could not perform consistent fetch, downgrading to fetching from CMS peers.", t);
+                //TODO (now): add back... it spams the logs making the real errors get truncated
```

Comment:
will add back, way too much spam in the test logs making it hard to find *real* issues...